vmm/src/device_manager.rs

```diff
@@ -1415,14 +1416,16 @@ impl DeviceManager {
         // Add virtio-rng if required
         let rng_config = self.config.lock().unwrap().rng.clone();
         if let Some(rng_path) = rng_config.src.to_str() {
+            let id = String::from(RNG_DEVICE_NAME_PREFIX);
+
             let virtio_rng_device = Arc::new(Mutex::new(
                 vm_virtio::Rng::new(rng_path, rng_config.iommu)
                     .map_err(DeviceManagerError::CreateVirtioRng)?,
             ));
             devices.push((
                 Arc::clone(&virtio_rng_device) as VirtioDeviceArc,
-                false,
-                None,
+                rng_config.iommu,
```

Comment:
Split this into two. One part to fix the iommu bug and then another to assign the name. For the commit message on the latter don't refer to "--rng" as that's confusing as that it a CLI command and this is independent of the CLI.

Reply:
Can you do the commit message fix too?